src/common/LogEntry.h

```diff
@@ -196,14 +202,14 @@ inline std::ostream& operator<<(std::ostream& out, const LogEntry& e)
 
 template <> struct fmt::formatter<EntityName> : fmt::formatter<std::string_view> {
   template <typename FormatContext>
-  auto format(const EntityName& e, FormatContext& ctx) {
-    return formatter<std::string_view>::format(e.to_str(), ctx);
+  auto format(const EntityName& e, FormatContext& ctx) const {
+    return fmt::formatter<std::string_view>::format(e.to_str(), ctx);
```

Comment:
I'd suggest moving the two formatters into the fmt namespace (and saving us a problem with
some compiler versions)

Reply:
ca16531f92be19075d3ca820adb3a0de95e393b6